starlette/concurrency.py

```diff
@@ -1,18 +1,24 @@
 import asyncio
 import functools
+import sys
 import typing
 from typing import Any, AsyncGenerator, Iterator
 
 try:
-    import contextvars  # Python 3.7+ only.
+    import contextvars  # Python 3.7+ only or via contextvars backport.
 except ImportError:  # pragma: no cover
     contextvars = None  # type: ignore
 
+if sys.version_info >= (3, 7):  # pragma: no cover
```

Comment:
(Nit) Should we instead try/except with a comment that the except path will be taken on 3.6?

Reply:
try/except ImportError has the downside of not being mypy-friendly. But happy to switch to that if that's more the standard we're going with.